src/duke/Parser.java

```diff
@@ -0,0 +1,337 @@
+package duke;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * This is an Action class that obtains a sentence as input that
+ * can be deciphered to create tasks in the duke.Duke system
+ *
+ * @author  Hsiao Jiet
+ * @version 1.0
+ * @since   2022-1-15
+ */
+
+public class Parser {
+    protected String userInput;
+    protected TaskList tasks;
+    protected String[] listOfUserInput;
+    protected Storage storage;
+    String line = "\n____________________________________________________________\n";
```

Comment:
Honestly found this funny and gave me a chuckle because of how apt this is. Good job!

Reply:
HAHAHA this is the best comment I've ever seen in my entire life